front/app/components/AddToBasketButton/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { getLatestRelevantPhase } from 'api/phases/utils';
+
+// typings
+import { IIdea } from 'api/ideas/types';
+import { IPhases } from 'api/phases/types';
+import { IProject } from 'api/projects/types';
+
+export const getParticipationContext = (
```

Comment:
Bit unclear to me what exactly this does? Is it returning the latest ideation context phase/project then, given a certain idea? If so I think we should rename this function so it's clearer.

Reply:
If it's a continuous project, it just returns the project. If it's a timeline project, it returns the 'latest relevant phase' that has a relation to the idea. It's just a bunch of code that was already in the component and that I took out to make it reusable... I agree that the name is not super clear, I'll try to think of a better name